gameofgame.py

```diff
@@ -25,49 +25,65 @@ def choose_country():
 # Function to check if the guess is correct
 def check_guess(guess, country):
     capital = countries_capitals[country]
-
     if guess.lower() == capital.lower():
-    return True
+        return True
     else:
-    return False
- main
+        return False
+
+# Initialize player scores
+player1_score = 0
+player2_score = 0
 
 # Main game loop
-    while True:
+while True:
     # Choose a random country
     country = choose_country()
 
     # Both Players take turns to play
-    #player1 goes first
+    # player 1 goes first
     print("Player 1's turn")
     player1_guess = input(f"What is the capital of {country}? ")
     
-    ##player2
-    print ("Player 2's turn")
+    # player 2 goes next
+    print("Player 2's turn")
     player2_guess = input(f"What is the capital of {country}? ")
 
-    ##Reveal the correct answer
+    # Reveal the correct answer
     capital = countries_capitals[country]
-    print (f"The correct answer is {capital}.")
-    
+    print(f"The correct answer is {capital}.")
 
-     ##check the guesses and print results
+    # Check the guesses and update the scores
     if check_guess(player1_guess, country) and check_guess(player2_guess, country):
         print("Both players guessed correctly!")
+        player1_score += 1
+        player2_score += 1
     elif check_guess(player1_guess, country):
         print("Player 1 guessed correctly!")
```

Comment:
Might want to think about how to refactor this message (i.e. defining the text on a string once and then reusing it below), so you can reuse this variable when printing the similar message for Player 2 below.